lib/Controller/LoginFlowController.php

```diff
@@ -155,9 +155,13 @@ public function login(): RedirectResponse {
 			$this->autoProvisioningService->updateAccountInfo($user, $userInfo);
 		}
 
-		// trigger login process
-		if ($this->userSession->createSessionToken($this->request, $user->getUID(), $user->getUID()) &&
-			$this->userSession->loginUser($user, null, OpenIdConnectAuthModule::class)) {
+		// trigger login
+
+		$tokenCreated = $this->userSession->createSessionToken($this->request, $user->getUID(), $user->getUID());
+		// @phpstan-ignore-next-line because loginUser does not have the third argument in older versions
+		$userLoggedIn = $this->userSession->loginUser($user, null, OpenIdConnectAuthModule::class);
```

Comment:
this changes the logic .....

Reply:
How?

Reply:
Do you mean the refactoring out of the if statement or calling loginUser with the third argument?

Reply:
Okay then back to the more ugly solution ;)